internal/cloud_connector/message_handlers_test.go

```diff
@@ -65,15 +66,17 @@ func TestHandleOnlineMessagesNoExistingConnection(t *testing.T) {
 	var connectedClientRecorder = &mockConnectedClientRecorder{}
 	var sourcesRecorder controller.SourcesRecorder
 
+	logger := logger.Log.WithFields(logrus.Fields{})
```

Comment:
I think we can remove this line and pass `logger.Log` into the `handleOnlineMessage()` function.